travis/deploy.sh

```diff
@@ -45,10 +58,14 @@ then
     sed -i 's/name="c2cgeoportal",/name="c2cgeoportal-win",/g' setup.py
     git diff
 
-    if [ ${FINAL} == true ]
+    if [ ${BUILD_TAG} != false ]
+    then
+        .build/venv/bin/python setup.py egg_info --no-date --tag-build "${BUILD_TAG}" sdist upload -r c2c-internal
+    else if [ ${FINAL} == true ]
     then
         .build/venv/bin/python setup.py egg_info --no-date --tag-build "" sdist upload -r c2c-internal
     else
         .build/venv/bin/python setup.py sdist upload -r c2c-internal
     fi
+    fi
```

Comment:
The double "if" comes from the newly added "else if"?
Shouldn't you add some indent to make the blocks clearer?